admin/class-monk-admin.php

```diff
@@ -124,45 +124,68 @@ public function monk_add_menu_page() {
 	 * Function to create a section for General Options in the administration menu
 	 *
 	 * @since    1.0.0
-	*/
+	 */
 	public function monk_options_init() {
-		register_setting( 'general_options', 'monk_language' );
 		add_settings_section(
 			'monk_general_options',
 			__( 'General options', 'monk' ),
 			array( $this, 'monk_settings_section_render'),
 			'general_options'
 		);
+
+		register_setting( 'general_options', 'monk_default_language' );
 		add_settings_field(
 			'monk_language_field',
 			__( 'Default language', 'monk' ),
 			array( $this, 'monk_language_select_render' ),
 			'general_options',
 			'monk_general_options'
 		);
+
+		register_setting( 'general_options', 'monk_active_languages' );
+		add_settings_field(
+			'monk_list',
```

Comment:
Don't use generic names as `monk_list` use something like `monk_active_languages`  and adjust where is needed.